meilisearch-http/src/index/search.rs

```diff
@@ -107,6 +108,7 @@ impl Index {
         };
 
         let result = SearchResult {
+            exhaustive_nb_hits: false, // not implemented, we use it to be ISO
```

Comment:
```suggestion
            exhaustive_nb_hits: false, // not implemented yet
```

Reply:
We should not talk about being ISO in the code since this code will be merged into MeiliSearch 🙂